README.md

```diff
@@ -2,4 +2,586 @@
+locale, which is specified as “zxx-­XX”. The ilib-env package has a `setLocale()` method that
+allows you to set the overall default locale for ilib which is used if no explicit locale
+is set in the parameters to a constructor or method.
+
+Example:
+
+```
+import { setLocale } from 'ilib-env';
+
+setLocale("zxx-­XX");
+```
+
+The “unknown” locale causes the `ResBundle` class to return strings that are pseudo­localized.
+That is, many of the regular Latin characters are replaced algorithmically with accented versions
+of those same base characters. This is still readable by an English­-speaking QA person,
+and yet they immediately know that the string was resourcified properly. If a string appears in
+plain text, then it is either user-­entered data, strings from 3rd party software or services,
+or it is a string that has not been resourcified properly. The QA person can submit a bug for
+the engineer to investigate which of those possibilities it is, and fix the code if it turns out
+to be a string that is not wrapped with a `getString()` method call.
+
+Here is what a pseudo­localized string looks like:
+
+```
+    "Greetings from Paris": "Ĝŕëëţíñğš fŕõm Pàŕíš"
+```
+
+Things get a little tricky if the string is intended to be formatted or used in a web page. In
+general, it is not a good idea to put HTML inside of a translatable string because the translators
+are linguists, not programmers, and they do not fully understand the syntax of HTML. They often
+translate things like CSS class names or HTML keywords. However, sometimes it is necessary to do this.
+
+For example, if you put a link in your string like this:
+
+```
+    "Greetings from <a href='url'>{city}</a> &amp; {country}"
+```
+
+Now if you just pseudotranslated every character, you would end up with a string like this:
+
+```
+    "Ĝŕëëţíñğš fŕõm <à ĥŕëf='üŕľ'>{çíţÿ}</à> &àmþ; {çõüñţŕÿ}"
+```
+
+Obviously, that would not work as intended in the browser, as the HTML tags and entities are
+also pseudotranslated, as are the replacement parameter names. In this case, you should instantiate
+your ResBundle with the `type` property set to “html”, and the psuedo­translation code will
+automatically parse the strings and skip any HTML tags or entities.
+
+In the examples above, we already have the `type` property set correctly. With the type
+parameter set, the above example would come out as this instead:
+
+```
+    "Ĝŕëëţíñğš fŕõm <a href='url'>{city}</a> &amp; {country}"
+```
+
+Note that the IString replacement parameters have been left alone as well. Without that, the
+substitution would not work too well either!
+
+The constructor also supports other types of strings: xml, html, text, c, raw, ruby, or template.
+
+See the full [API reference documentation](./docs/ilibResBundle.md) for a full description of
+all the parameters to the ResBundle and String classes.
+
+## Samples
+
+There are some simple sample projects available that illustrate using `ResBundle`, `loctool`,
```

Comment:
maybe add links to directly visit project?

Reply:
Oh yes, forgot to do that. Also, the webos one does not use ilib-resbundle, so I'll remove it

Reply:
I forgot to commit and make a PR out of the sample changes, so I just did that one: https://github.com/iLib-js/ilib-loctool-samples/pull/31